src/index.js

```diff
@@ -69,17 +74,33 @@ function loginUser() {
     domUpdates.hide('.login-form');
   } else if (arrayOfFifty.includes(userInput) &&
     passwordInput === 'travel2020') {
+    traveler = new Traveler(extractUserId(userInput))
     domUpdates.hide('.login-page');
     domUpdates.displayTraveler('.traveler-dashboard')
     populateDestinationSelect(destinationData)
+    domUpdates.populateTravelerInfo(traveler, traveler.id, tripData, destinationData)
+    domUpdates.populateUpcomingTrips(traveler, traveler.id, tripData, destinationData)
+    domUpdates.populatePastTrips(traveler, traveler.id, tripData, destinationData)
+    domUpdates.populatePendingRequests(traveler, traveler.id, tripData, destinationData)
   } else {
     $('.login-error').text(`Please enter in a valid user name and login`)
   }
 }
 
+//helper function in login
 function populateDestinationSelect(destinationData) {
   destinationData.forEach(destination => {
     domUpdates.populateDestinationDropDown(destination)
   })
 }
 
+//helper function in login
+function extractUserId(userInput) {
+  if (userInput.length === 10) {
+    let id = userInput.split('').splice(-2, 2).join('');
+    return userData.find(user => user.id === parseInt(id))
```

Comment:
You should only need to return one time at the end of this function (you have duplicate code on 101 and 104, so you should be able to just have the one return statement after the curly on line 105

Reply:
Hmmmm I think I see what youre saying here. Establish the variable id and return the statement at the end based upon the conditional